.flowconfig

```diff
@@ -7,6 +7,7 @@
 
 [libs]
 ./node_modules/fusion-core/flow-typed
+./node_modules/fusion-plugin-i18n/flow-typed
```

Comment:
Consumers would have to add this to their `.flowconfig` to include the `libdef` for [`locale`](https://github.com/fusionjs/fusion-plugin-i18n/blob/master/flow-typed/locale.js).  Without this, imports would be of type `any` (and are consequently uncovered).